<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSearchPrestamo</class>
 <widget class="QWidget" name="FormSearchPrestamo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblFiltro">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville Old Face</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Filtrar: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxFiltro">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Todas</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cota</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cédula</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fecha Préstamo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fecha Entrega</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Responsable</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditFiltro">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFiltrar">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Filtrar</string>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/images/ver.png</normaloff>:/images/ver.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
